packages/eslint-config-appium/package.json

```diff
@@ -28,7 +28,7 @@
     "eslint:find:unused-rules": "eslint-find-rules -u -n",
     "lint": "eslint index.js"
   },
-  "dependencies": {
+  "peerDependencies": {
```

Comment:
this sucks. i wonder why we got away with this for so long.

Reply:
dumb luck, I guess.